tests/test_content_client.py

```diff
@@ -0,0 +1,169 @@
+import pytest
+import testinfra
+import time
+import subprocess
+
+
+@pytest.fixture
+def katello_client(request, entities, registration_hostname):
+    image_id = subprocess.check_output(['podman', 'images', '-q', '--filter', f"reference={request.param}"])
+    assert image_id
+
+    try:
+        container_id = subprocess.check_output([
+            'podman',
+            'run',
+            '--volume',
+            '/dev/log:/dev/log',
+            '--detach',
+            '--tty',
+            request.param
+        ]).decode().strip()
```

Comment:
This should be outside the `try`. If this statement fails, `container_id` is not defined and `finally` will fail on a `NameError` exception.